includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -283,7 +283,9 @@ public static function register() {
 					'manage_terms' => AMP_Validation_Manager::VALIDATE_CAPABILITY, // Needed to give access to the term list table.
 					'delete_terms' => AMP_Validation_Manager::VALIDATE_CAPABILITY, // Needed so the checkbox (cb) table column will work.
 					'assign_terms' => 'do_not_allow', // Block assign_terms since associating terms with posts is done programmatically.
-					'edit_terms'   => 'do_not_allow', // Terms are created (and updated) programmatically.
+					// Terms are created (and updated) programmatically, but we still need to assign capabilities while generating edit link for term.
+					// @see <https://github.com/ampproject/amp-wp/issues/7604#issuecomment-1704244763>.
+					'edit_terms'   => AMP_Validation_Manager::VALIDATE_CAPABILITY,
```

Comment:
Won't this then allow users to access the edit term page when they shouldn't be able to access it?

Reply:
At the very least we'll also need to `unset($actions['edit'])` in:

https://github.com/ampproject/amp-wp/blob/828f229af54b9462a9e96152530a920d7431b1ca/includes/validation/class-amp-validation-error-taxonomy.php#L1679-L1738

Then there's the question of the REST API. I suppose they can't access it because it's not `public`  and `show_in_rest` is not `true`.

On the other hand, is this actually a bug that should rather be fixed in core?

Reply:
> Won't this then allow users to access the edit term page when they shouldn't be able to access it?

We are not providing an edit link so it will be hard to generate the link with the correct `tag_ID`.

> At the very least we'll also need to unset($actions['edit']) in:

I think it will do it.

https://github.com/ampproject/amp-wp/blob/828f229af54b9462a9e96152530a920d7431b1ca/includes/validation/class-amp-validation-error-taxonomy.php#L1733

Reply:
> On the other hand, is this actually a bug that should rather be fixed in core?

Yes, and I have filed a ticket for that - https://core.trac.wordpress.org/ticket/59336